test/utils.ts

```diff
@@ -14,7 +14,7 @@ export function setupServerAndListen(
   ...handlers: Array<RequestHandler>
 ): SetupServer {
   const server = setupServer(...handlers)
-  server.listen({ onUnhandledRequest: 'error' })
+  server.listen({ onUnhandledRequest: 'warn' })
```

Comment:
What other options do we have?
Can we rewrite the test or use another tool to avoid working around it?

Reply:
here is the options 

"warn" (Default) | Print a warning but perform the request as-is.
-- | --
"error" | Print an error and halt request execution.
"bypass" | Does not print anything and perform the request as-is.

======================

If we want to keep using the `error` option, We could update the `flush` test case @cre8ivejp

Reply:
I restore the code https://github.com/bucketeer-io/javascript-client-sdk/pull/187/commits/be3d64165249ef3d2a40ae8fb27480f2d3a01129

The unhandled request was created because the `cross-fetch` was not working as expected.
We don't need to change any.

Reply:
Nice catch!